docs/devcontainers/create-dev-container.md

```diff
@@ -49,7 +49,7 @@ For this example, if you'd like to install the [ESLint extension](https://market
 
 ```json
 {
-    "image": "mcr.microsoft.com/devcontainers/typescript-node:0-12",
+    "image": "mcr.microsoft.com/devcontainers/typescript-node:0-18",
```

Comment:
Same comment here - it might be worth highlighting image metadata here and using a different extension other than eslint.

Reply:
Just made the same updates here!

Reply:
Also linking to the more detailed pre-build and label info here now.